test/integration/default/serverspec/groups_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'serverspec_helper'
+
+describe 'nexus::groups' do
+  if os[:family] == 'windows'
+    auth_info = "$base64AuthInfo = [Convert]::ToBase64String([Text.Encoding]::ASCII.GetBytes((\"{0}:{1}\" \
+-f 'admin','admin123')));"
+    base_uri = '-URI http://localhost:8081/service/rest/v1/script'
+
+    run_get_group = "#{auth_info} Invoke-RestMethod -Headers @{Authorization=(\"Basic {0}\" -f $base64AuthInfo)} \
+#{base_uri}/get_repo/run -Method POST -ContentType 'text/plain' -Body '%{group}'"
+
+    describe command("powershell -command { #{run_get_group} }" % { group: 'foor-group' }) do
```

Comment:
how come it's "foor-group" (typo?) and the tests pass?

Reply:
test is on linux :)